metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -464,7 +464,17 @@ def process_schemas(self, catalog: Catalog) -> Iterable[MetadataWorkUnit]:
 
             with self.report.new_stage(f"Ingest schema {schema.id}"):
                 yield from self.gen_schema_containers(schema)
-                yield from self.process_tables(schema)
+                try:
+                    yield from self.process_tables(schema)
+                except Exception as e:
+                    logger.exception(f"Error parsing schema {schema}")
+                    self.report.report_warning(
+                        message="schema-parse",
+                        context=str(schema),
+                        title=f"Error parsing schema {schema}",
```

Comment:
AFAIK `title` is expected to be a literal, free of vars, for the grouping

```suggestion
                        title="Error parsing schema",
                        message="Missed schema because of parsing issues",
                        context=str(schema),
```